spec/calendar.html

```diff
@@ -8,14 +8,14 @@ <h1>Temporal.Calendar Objects</h1>
   <emu-clause id="sec-temporal-calendar-abstract-ops">
     <h1>Abstract Operations for Temporal.Calendar Objects</h1>
 
-    <emu-clause id="sec-temporal-createtemporalcalendarfromstatic" aoid="CreateTemporalCalendarFromStatic">
-      <h1>CreateTemporalCalendarFromStatic ( _constructor_, _identifier_ )</h1>
+    <emu-clause id="sec-temporal-createtemporalcalendar" aoid="CreateTemporalCalendar">
+      <h1>CreateTemporalCalendar ( _identifier_ [ , _newTarget_ ] )</h1>
       <emu-alg>
         1. Assert: ! IsBuiltinCalendar(_identifier_) is *true*.
-        1. If ! IsConstructor(_constructor_) is *false*, throw a *TypeError* exception.
-        1. Let _result_ be ? Construct(_constructor_, « _identifier_ »).
-        1. Perform ? RequireInternalSlot(_result_, [[InitializedTemporalCalendar]]).
-        1. Return _result_.
+        1. If _newTarget_ is not given, set it to %Temporal.Calendar%.
```

Comment:
```suggestion
        1. If _newTarget_ is not provided, set it to %Temporal.Calendar%.
```
i think this is the phrasing usually used